### PR TITLE
Question re: disabilities open to multiple members

### PR DESCRIPTION
**Pull request by jayroh**
When asking a household of multiple members the question regarding
disabilities will now ask IF and WHO is disabled in the household.

### CODE REVIEW

**Comment by jayroh** [comment]
[[Medicaid MM Flow] Quick Health Questions: Does anyone have a disability? + Nested](https://trello.com/c/Nd5VajaP/436-medicaid-mm-flow-quick-health-questions-does-anyone-have-a-disability-nested)

File: app/controllers/medicaid/health_disability_member_controller.rb [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+    def skip?
+      no_one_disabled? || single_member_household?
+    end
+
+    def member_attrs
+      %i[disabled]
+    end
+
+    def no_one_disabled?
```
Comment (jessieay) [inline app/controllers/medicaid/health_disability_member_controller.rb]: `nobody_disabled` ?

File: app/steps/medicaid/health_disability.rb [hunk 2]
```diff
@@ -2,6 +2,6 @@
 
 module Medicaid
   class HealthDisability < Step
-    step_attributes(:disabled)
+    step_attributes(:anyone_disabled)
```
Comment (jessieay) [inline app/steps/medicaid/health_disability.rb]: hooray!

File: app/steps/medicaid/health_disability_member.rb [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+module Medicaid
+  class HealthDisabilityMember < ManyMembersStep
+    step_attributes(:members)
+
+    def valid?
+      if members.select(&:disabled).any?
+        true
+      else
+        errors.add(:disabled, "Make sure you select at least one person")
```
Comment (jessieay) [inline app/steps/medicaid/health_disability_member.rb]: we have this error string a lot, wonder if we should throw it in i18n for share-ablity at some point

File: app/steps/medicaid/step_navigation.rb [hunk 4]
```diff
@@ -23,6 +23,7 @@ class StepNavigation
       ],
       "Health" => [
         Medicaid::HealthDisabilityController,
+        Medicaid::HealthDisabilityMemberController,
```
Comment (jessieay) [inline app/steps/medicaid/step_navigation.rb]: ❤️  love we that we have this pattern going - feels soooo predictable

File: db/migrate/20171026150119_rename_disabled_to_anyone_disabled.rb [hunk 5]
```diff
@@ -0,0 +1,19 @@
+class RenameDisabledToAnyoneDisabled < ActiveRecord::Migration[5.1]
+  def up
+    add_column :medicaid_applications, :anyone_disabled, :boolean
```
Comment (jessieay) [inline db/migrate/20171026150119_rename_disabled_to_anyone_disabled.rb]: should we default to `false` ? can't remember if we do that for other similar fields

File: app/controllers/medicaid/health_disability_controller.rb [hunk 6]
```diff
@@ -2,5 +2,16 @@
 
 module Medicaid
   class HealthDisabilityController < MedicaidStepsController
+    private
+
+    def after_successful_update_hook
+      if single_member_household?
```
Comment (jessieay) [inline app/controllers/medicaid/health_disability_controller.rb]: should we have a  controller spec for this condition?

File: db/migrate/20171026150119_rename_disabled_to_anyone_disabled.rb (hunk 5 above)
Comment (jayroh) [reply db/migrate/20171026150119_rename_disabled_to_anyone_disabled.rb]: I don't think we should and here's why - if we have `nil` for these answers we know they have not been answered. Whereas a false or true will tell us they've been answers. I think it's important to be able to infer if a client has gone through/by that step and if we default to a value, we'd lose that visibility.

File: app/steps/medicaid/health_disability_member.rb (hunk 3 above)
Comment (jayroh) [reply app/steps/medicaid/health_disability_member.rb]: Yeah I think it's a good idea. Worthwhile to go in the backlog:

https://trello.com/c/SvkqRfzJ/500-pull-make-sure-you-select-at-least-one-person-out-from-the-controllers-and-into-locales-enyml

**Pull request merged**